package.json

```diff
@@ -6,19 +6,21 @@
   "homepage": "https://github.com/CityOfZion/neon-wallet",
   "author": "Ethan Fast <ejhfast@gmail.com> (https://github.com/Ejhfast)",
   "scripts": {
-    "postinstall": "electron-builder install-app-deps",
+    "postinstall": "electron-builder install-app-deps && electron-rebuild --force --module_dir . -w node-hid -v 1.7.9",
     "start": "electron .",
+    "start-dev": "electron . --enable-logging --remote-debugging-port=9222",
     "assets": "webpack --config ./config/webpack.config.prod",
     "assets-dev": "webpack --config ./config/webpack.config.dev",
     "assets-watch": "node_modules/.bin/webpack-dev-server --config ./config/webpack.config.dev  --port 3000 --host 0.0.0.0 --inline",
-    "test": "jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "test": "yarn add electron@1.7.9 && jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
```

Comment:
npm rebuild node-hid instead of yarn add electron@1.7.9 gets the tests to pass. since it is just for the tests, lets use npm here unless someone can figure out a solution that works with yarn. @shawnmclean what do you think? There is yarn install --force which has been deprecated for yarn add, which is causing the tests to fail.

also, @coranos there is an issue with one of the tests. I know what the problem is, but lets not fix it yet, since I think more changes will be coming that might require that to change further. We can get all the tests to pass as the last step

Reply:
note that one issue with this solution is that you need to run yarn after running yarn test or else yarn start does not work

Reply:
last comment on this is that I tried mocking node-hid in the test files, but it did not seem to bypass the module errors. maybe I was not mocking it in a way that solved the issue correctly. I created a file for node-hid in __mocks__ with HID and devices as jest functions and then called jest.mock('node-hid') in the test files